### DISCUSSION TITLE
멀티플레이 씬 전환 후 라이트 세팅 관련

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#721] [5. Q&A]
월드 1.3.0 입니다 

멀티플레이에서 씬 전환후 라이트 세팅했는데 까마케 나오는데 해결 방법 없나요?

![image](https://user-images.githubusercontent.com/50188795/167420870-4356fc9f-e83b-4764-8405-963e0dc65674.png)

아래처럼 수동을해 주면 잘 나옵니다...
![image](https://user-images.githubusercontent.com/50188795/167420911-ec0d15b7-cc7c-4d1c-8f60-693b74a14ed8.png)

![image](https://user-images.githubusercontent.com/50188795/167420924-f112d18e-07bd-478d-b9f2-2ec7f7b2e548.png)

https://user-images.githubusercontent.com/50188795/167420944-9840e7a2-13bf-4d8b-8b2a-a6d72ab87705.mp4

https://user-images.githubusercontent.com/50188795/167420961-8665edc5-12b2-4a41-ada1-e9d2a0f41770.mp4

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, #515 와 동일한 이슈로 확인 작업 진행 중입니다.
해결 전까지 불편하시겠지만 Set Scene Lighting으로 부탁 드립니다.
감사합니다.
